src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -141,9 +141,8 @@
   public static InstantiatingChannelProvider.Builder defaultChannelProviderBuilder() {
     return InstantiatingChannelProvider.newBuilder()
         @if xsettingsClass.hasDefaultServiceAddress
-            .setServiceAddress(DEFAULT_SERVICE_ADDRESS)
+            .setEndpoint(DEFAULT_SERVICE_ADDRESS + ':' + DEFAULT_SERVICE_PORT)
```

Comment:
I would prefer to also change the default constants to use just one - DEFAULT_ENDPOINT.